website/source/api/agent/service.html.md

```diff
@@ -308,37 +308,6 @@ curl localhost:8500/v1/agent/health/service/name/web
             "Meta": null,
             "Port": 80,
             "EnableTagOverride": false,
-            "ProxyDestination": "",
-            "Connect": {
-                "Native": false,
-                "Proxy": null
-            },
-            "CreateIndex": 0,
-            "ModifyIndex": 0
-        }
-    ],
-    "passing": [
```

Comment:
I think you nuked 312->340 by accident.

Reply:
Opened https://github.com/hashicorp/consul/pull/6303